tests/resources/assessments/pipeline_config.yml

```diff
@@ -26,3 +26,6 @@ steps:
     mode: overwrite
     frequency: daily
     flag: active
+    dependencies:
+      - pandas
+      - duckdb
```

Comment:
Maybe we can add a test for a dependency with a version specified as well?

Reply:
Let me check.